doc/source/parameters/cfd/boundary_conditions_cfd.rst

```diff
@@ -5,6 +5,7 @@ This subsection defines the boundary conditions associated with fluid dynamics p
 
 * ``noslip`` boundary conditions strongly impose the velocity on a boundary to be :math:`\mathbf{u}=[0,0]^T` and :math:`\mathbf{u}=[0,0,0]^T` in 2D and 3D respectively.
 * ``slip`` boundary conditions impose :math:`\mathbf{u} \cdot \mathbf{n}=0`, with :math:`\mathbf{n}` the normal vector of the boundary. Imposing slip boundary conditions strongly is not trivial in FEM. We refer the reader to the deal.II `documentation <https://www.dealii.org/current/doxygen/deal.II/group__constraints.html>`_ for explanations on how this is achieved.
+* ``partial slip`` boundary condition introduces the "penalization factor" ``beta`` to the:math:`\mathbf{n}` normal vector of the boundary, and the ``boundary layer thickness`` (m) as a parameter to calculate the shear stress at the boundaries.
```

Comment:
```suggestion
* ``partial slip`` boundary condition introduces the "penalization factor" ``beta`` to the :math:`\mathbf{n}` normal vector of the boundary, and the ``boundary layer thickness`` (m) as a parameter to calculate the shear stress at the boundaries.
```

Reply:
I don't think I would talk about the beta parameter before introducing the equation. I think I would just say that you and manually changing the size of the boundary layer.